client-src/elements/chromedash-userlist.ts

```diff
@@ -93,21 +91,30 @@ class ChromedashUserlist extends LitElement {
   // TODO(jrobbins): Change this to be a JSON API call via csClient.
   async ajaxSubmit(e) {
     e.preventDefault();
-    const formEl = this.shadowRoot.querySelector('form');
+    const formEl = this.renderRoot.querySelector('form')!;
 
     if (formEl.checkValidity()) {
-      const email = formEl.querySelector('input[name="email"]').value;
-      const isAdmin = formEl.querySelector('input[name="is_admin"]').checked;
-      const isSiteEditor = formEl.querySelector(
+      const emailInput = formEl.querySelector(
+        'input[name="email"]'
+      ) as HTMLInputElement;
+      const adminCheckbox = formEl.querySelector(
+        'input[name="is_admin"]'
+      ) as SlCheckbox;
+      const siteEditorCheckbox = formEl.querySelector(
         'input[name="is_site_editor"]'
-      ).checked;
+      ) as SlCheckbox;
+      const email = emailInput?.value;
+      const isAdmin = adminCheckbox?.checked;
+      const isSiteEditor = siteEditorCheckbox?.checked;
       window.csClient.createAccount(email, isAdmin, isSiteEditor).then(json => {
         if (json.error_message) {
           alert(json.error_message);
         } else {
           this.addUser(json);
           formEl.reset();
-          formEl.querySelector('input[name="is_site_editor"]').disabled = false;
```

Comment:
Don't move these queries out of a callback, because the DOM can change between the call to `createAccount` and the execution of the stuff in `.then()`. It _might_ not in this case, depending on what's in `render()`, but it's safest to just keep the query in its original location.